alma/lib/Utils/functions.php

```diff
@@ -150,15 +150,31 @@ function almaFormatPrice($cents, $id_currency = null)
 function getDateFormat($locale, $timestamp)
 {
     try {
+        if (!class_exists(IntlDateFormatter::class)) {
+            return getFrenchDateFormat($timestamp);
+        }
         $formatter = new IntlDateFormatter($locale, IntlDateFormatter::MEDIUM, IntlDateFormatter::NONE);
         if ($formatter === null) {
             throw new IntlException(intl_get_error_message());
         }
 
         return $formatter->format($timestamp);
-    } catch (IntlException $e) {
-        $date = new DateTime($timestamp);
-
-        return $date->format('m/d/Y');
+    } catch (Exception $e) {
+        return getFrenchDateFormat($timestamp);
     }
 }
+
+/**
+ * fallback for when IntlDateFormatter is not available
+ *
+ * @param string $timestamp
+ *
+ * @return string
+ */
+function getFrenchDateFormat($timestamp)
+{
+    $date = new DateTime();
+    $date->setTimestamp($timestamp);
+
+    return $date->format('d/m/Y');
```

Comment:
previous format was `m/d/Y` ... is it assumed to always fallback on french date format ?

Reply:
Most of our merchant are French and our guess would be that international merchants have more incentive to have php-intl installed on their server, so having a french fallback made more sense